client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -23,6 +23,12 @@ function filesToMarkdown(files = {}) {
   }, '\n');
 }
 
+function transformEditorLink(url) {
+  return url
+    .replace(/(?<=\/\/)([^.]+)(\.)([^.]+)(\.)(repl)(\.co)(\/?)/, '$5.it/@$3/$1')
+    .replace(/(?<=\/\/)([^.]+)(\.)(glitch)(\.me)(\/?)/, '$3.com/edit/#!/$1');
```

Comment:
Maybe use named capture groups instead of having those numbers?
```suggestion
    .replace(/(?<=\/\/)(?<projectname>[^.]+)(\.)(?<username>[^.]+)(\.)repl(\.co)(\/?)/, 'repl.it/@$<username>/$<projectname>')
    .replace(/(?<=\/\/)(?<projectname>[^.]+)(\.)glitch(\.me)(\/?)/, 'glitch.com/edit/#!/$<projectname>');
```